smile_id_core/Utilities.py

```diff
@@ -247,50 +247,6 @@ def validate_id_params(
                 raise ValueError(f"key {field} cannot be empty")
             else:
                 raise ValueError(f"key {field} cannot be empty")
-        if not use_validation_api:
```

Comment:
We should also remove `use_validation_api` from the function signature (and there fore update the changelog and bump the version?)

Reply:
We can deprecate it for now and remove it later in a later version

Reply:
You're right, we should remove it entirely